volatility3/framework/configuration/requirements.py

```diff
@@ -593,13 +593,14 @@ def unsatisfied(
     def matches_required(
         cls, required: Tuple[int, ...], version: Tuple[int, int, int]
     ) -> bool:
-        if len(required) > 0 and version[0] != required[0]:
-            return False
-        if len(required) > 1 and version[1] < required[1]:
-            return False
-        return True
+        versionutils.matches_required(required, version)
```

Comment:
```suggestion
        return versionutils.matches_required(required, version)
```
Subtle but... breaks everything if you don't return it...  5;P

Reply:
Thank you! I was really scratching my head this morning.